.env

```diff
@@ -1,6 +1,5 @@
-REACT_APP_SUPPORTED_ETHEREUM_NETWORKS="polygon,mumbai,mainnet,kovan,fuji,avalanche"
-REACT_APP_DEFAULT_ETHEREUM_NETWORK="mainnet"
 REACT_APP_ENABLE_CACHING_BACKEND="true"
```

Comment:
@sendra / @aschmidt20  this pr is NOT ready for review, but would <3 to get sme feedback on that parameter.

Should it be removed? don't really get why it exists as the existence of caching api in network config determines if it will be enabled or not. Also it's weird that it's all or nothing. Would just remove it :man_shrugging: